tests/lvmo/test_lvm_clone_bigger_than_disk.py

```diff
@@ -0,0 +1,219 @@
+import logging
+
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import tier1, skipif_lvm_not_installed
+from ocs_ci.framework.testlib import skipif_ocs_version, ManageTest, acceptance
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.cluster import LVM
+from ocs_ci.ocs.exceptions import LvSizeWrong
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
+@acceptance
```

Comment:
since this does not test the basic functionality. 
This seems like tier2.
Also, should not have the `acceptance` mark

Reply:
Added tier2, removed tier1 and acceptance.